katharsis-core/src/test/java/io/katharsis/queryspec/repository/QuerySpecResourceRepositoryTest.java

```diff
@@ -15,27 +15,27 @@
 
 public class QuerySpecResourceRepositoryTest extends AbstractQuerySpecTest {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add some tests to this class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2187)